meta/parse.pl

```diff
@@ -1247,6 +1247,10 @@ sub ProcessDefaultValue
     {
         WriteSource "$val = { .chardata = { 0 } };";
     }
+    elsif ($default =~ /^0\.0\.0\.0$/ and $type =~ /^(sai_ip4_t)/)
```

Comment:
why use explicit ipv4 instead of sai_ip_address_t? which is already supported on line 1236

Reply:
Accidental merge. I will be doing rebase